addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -176,10 +176,11 @@ class Network {
      * not setting it will remove the track from the server
      */
     async updateUpload(type, track) {
-        await Promise.all([
-            this.p2p.updateUpload(type, track),
-            this.sfu?.updateUpload(type, track),
-        ]);
+        const proms = [this.p2p.updateUpload(type, track)];
+        if (this.sfu?.state === "connected") {
+            proms.push(this.sfu.updateUpload(type, track));
+        }
+        await Promise.all(proms);
```

Comment:
This change isn't motivated in commit message. Would be nice to be silently handled in `sfu` kind of like `p2p` service, to keep similar API as much as possible.

Reply:
👍 it is in the commit message but I forgot to add it to the PR message, I will add it to the next SFU patch too so we can remove the check later